src/index.ts

```diff
@@ -106,9 +115,7 @@ getAllFiles(TEMPORARY_SCHEMA_DIR).then((fileInfoList) => {
 // run typescript to json schema
 }).then((schemaFileList) => {
 
-  const allSchemaPromises = [];
-
-  schemaFileList.forEach((schemaFilePath) => {
+  const allSchemaPromises = schemaFileList.map((schemaFilePath) => {
```

Comment:
mapping allows typescript to infer the type of all schema promises

Reply:
ooh, that's a nice tip. Love it.